src/streamlit_extras/pdf_viewer/__init__.py

```diff
@@ -0,0 +1,127 @@
+from __future__ import annotations
+
+import io
+from pathlib import Path
+from typing import Any, Union
+
+import numpy.typing as npt
+import streamlit as st
+
+from .. import extra
+
+PdfData = Union[
+    str,
+    Path,
+    bytes,
+    io.BytesIO,
+    io.RawIOBase,
+    io.BufferedReader,
+    "npt.NDArray[Any]",
```

Comment:
What about this? I don't see it handled explicitly below.

Reply:
yep, same here. Unnecessary copy from Streamlit codebase, removed it. I don't think it makes sense to support numpy array for pdf files, it could represent bytes but I think almost nobody would use it that way.